int32 num_gaps
std_msgs/Float64[] gap_size
geometry_msgs/Vector3[] gap_centers
